File: surf-cloud-standalone/src/main/java/dev/slne/surf/cloud/standalone/test/TestPacket.java
package dev.slne.surf.cloud.standalone.test;


import dev.slne.surf.cloud.standalone.redis.RedisEvent;
import lombok.Getter;

public class TestPacket extends RedisEvent {

  public static final String CHANNEL = "surf:test:channel";

  @Getter
  private TestEntity entity;

  public TestPacket() {
  }

  public TestPacket(TestEntity entity) {
    super(CHANNEL);
    this.entity = entity;
  }
}

File: surf-cloud-standalone/src/main/kotlin/dev/slne/surf/cloud/standalone/test/TestPacket.kt
package dev.slne.surf.cloud.standalone.test

import dev.slne.surf.cloud.standalone.redis.RedisEvent


class TestPacket : RedisEvent {
    var entity: TestEntity? = null
        private set

    constructor()

    constructor(entity: TestEntity?) : super(CHANNEL) {
        this.entity = entity
    }

    companion object {
        const val CHANNEL: String = "surf:test:channel"
    }
}
